acct-group/render/render-0-r1.ebuild

```diff
@@ -0,0 +1,8 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+ACCT_GROUP_ID=28
```

Comment:
In the systemd ebuild it was 30. Here it's 28. Can this be a problem?

Reply:
Yes, this could raise problems, there are few groups for which we use a different gid.